### ISSUE TITLE
UI: fix margin left

### ISSUE THREAD

**Issue by jim4067** [ironforge-cloud/soldev-ui#1]
the left side margin on my laptop is messed up, especially when filtering

![image](https://user-images.githubusercontent.com/30603522/223482197-5ed8f073-e107-4ef6-8636-9b4b00f962ef.png)

**Comment by cdhiraj40** [comment]
Hey @jim4067, thanks for letting us know! Can you update your issue with the below template?

```**Describe the bug:**
<!-- A clear and concise description of what the bug is. -->

**Steps to reproduce the behavior:**
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See the error

**Expected behavior:**
<!-- A clear and concise description of what you expected to happen. -->

**Screenshots:**
<!-- If applicable, add screenshots to help explain your problem. -->

**Additional context:**
<!--Add any other context about the problem here. -->```

**Comment by jim4067** [comment]
@cdhiraj40. Fixed in #5
